check_python.sh

```diff
@@ -116,8 +116,8 @@ pushd "${PROJECT_REPO}" > /dev/null || exit 1
     fi
     if [[ ${script_file} == *.py ]]; then
       # if there is a shebang use it.
-      if grep '^#!/bin/.*python' "${script_file}"; then
-        if grep '^#!/bin/.*python3' "${script_file}"; then
+      if grep '^#!(/usr)*/bin/.*python' "${script_file}"; then
+        if grep '^#!(/usr)*/bin/.*python3' "${script_file}"; then
```

Comment:
Aren't you really wanting to constrain this test to the first line in the file and then you can probably relax the pattern a bit.

That aside, IIRC `(...)` is ERE syntax and probably needs the `-E` argument.  Also, I think you want `?` to match 0 or 1 occurrence rather than `*`'s 0 or more occurrences.

Reply:
Changed in next push.